addon/src/wait-for-fetch.ts

```diff
@@ -0,0 +1,27 @@
+import { default as waitForPromise } from './wait-for-promise';
+
+export async function waitForFetch(fetchPromise: ReturnType<typeof fetch>) {
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  waitForPromise(fetchPromise);
+
+  const response = await fetchPromise;
```

Comment:
```suggestion
  const response = await waitForPromise(fetchPromise);
```

Reply:
The eslint-disable weirded me out, when there's no need to write it this way anyway.